src/main/java/com/epam/esm/controller/AuthController.java

```diff
@@ -0,0 +1,53 @@
+package com.epam.esm.controller;
+
+import com.epam.esm.domain.Role;
+import com.epam.esm.domain.User;
+import com.epam.esm.dto.LoginRequestDTO;
+import com.epam.esm.security.AppUserDetailsManager;
+import com.epam.esm.util.JwtTokenUtil;
+import jakarta.servlet.http.HttpServletResponse;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Set;
+
+@RestController
+@RequestMapping("api/public")
+@RequiredArgsConstructor
+public class AuthController {
+    private final AuthenticationManager authenticationManager;
+    private final JwtTokenUtil jwtTokenUtil;
+    private final AppUserDetailsManager userDetailsManager;
+    private final PasswordEncoder passwordEncoder;
+
+    @PostMapping("login")
```

Comment:
is it works? please update this to /login because there are a reason why this can't work properly

Reply:
yes, it works) but I changed to /login and /signup